src/wallet/feebumper.cpp

```diff
@@ -35,11 +35,8 @@ static int64_t CalculateMaximumSignedTxSize(const CTransaction &tx, const CWalle
         assert(mi != wallet->mapWallet.end() && input.prevout.n < mi->second.tx->vout.size());
         vCoins.emplace_back(CInputCoin(&(mi->second), input.prevout.n));
     }
-    if (!wallet->DummySignTx(txNew, vCoins)) {
-        // This should never happen, because IsAllFromMe(ISMINE_SPENDABLE)
-        // implies that we can sign for every input.
-        return -1;
-    }
+    // IsAllFromMe(ISMINE_SPENDABLE) implies that we can sign for every input.
+    assert(wallet->DummySignTx(txNew, vCoins));
```

Comment:
While I think the comment here is correct, I don't think that adding an assert is very helpful to detecting if this were to change -- having the software crash on some user in the wild if somehow we violate this assumption in some edge case seems worse than recovering and producing an error message.

Reply:
Side note, I should split in two lines as per @MarcoFalke suggestion.

Reply:
Maybe we should remove the comment.